WordPress/src/main/res/values/strings.xml

```diff
@@ -935,6 +935,8 @@
     <string name="error_refresh_unauthorized_comments">You don\'t have permission to view or edit comments</string>
     <string name="error_refresh_unauthorized_pages">You don\'t have permission to view or edit pages</string>
     <string name="error_refresh_unauthorized_posts">You don\'t have permission to view or edit posts</string>
+    <string name="error_refresh_unauthorized_profile">You don\'t have permission to view or edit your profile - check your configured site is whitelisted</string>
+    <string name="error_refresh_profile_not_loaded">There was an error trying to access your profile information</string>
```

Comment:
Can this be shortened? Something like "Can't access your profile information" would work.